modules/evm/src/tests.rs

```diff
@@ -1925,6 +1925,7 @@ fn remove_account_with_provides_should_panic() {
 }
 
 #[test]
+#[should_panic(expected = "removed account while is still linked to contract info")]
```

Comment:
This doesn't look correct. Ref count is 1 so calling remove_account will make it zero resulting into account removal

Reply:
we have `remove_contract` to remove the contract account. `remove_account` is only for removing a normal account. So no matter whether there is a ref or not, it should report an error in the develop mode